### DISCUSSION TITLE
Language Layout Change Support Threshold

### DISCUSSION THREAD

**Discussion by flide** [8VIM/8VIM#344] [Polls]
Recently, there have been a few PR's to change layouts for the languages. I am hesitant to merge these PR's as a change in the layout forces all the users for the language to relearn the keyboard.

**Comment by kjoetom** [comment]
I'd tend to answer 3, but we'd have to make sure we don't get a bias that more readily available layouts (e.g. because already included in 8vim) are used more than others that have to be downloaded and manually installed.
